.github/CODEOWNERS

```diff
@@ -100,7 +100,7 @@
 /tools/linting/* @thomaspink @lukasholzer
 /tools/linting/src/** @rowa-audil @lara-aigmueller @heartdisease
 
-/tools/release/** @thomaspink
+/tools/release/** @lukasholzer
```

Comment:
please add at least a second person. Having only one owner is potentially blocking us from moving forward and will bottleneck and silo us.

Reply:
good point will add @ffriedl89  who did the refactoring